test/modules/linear_boltzmann_solvers/transport_steady/transport_3d_5_cycles_2_restart.lua

```diff
@@ -0,0 +1,130 @@
+-- 3D Transport test with Vacuum BCs.
+-- SDM: PWLD
+-- Test: Max-value1=6.55387e+00
+--       Max-value2=1.02940e+00
+
+num_procs = 4
+
+--############################################### Check num_procs
```

Comment:
Could you remove the `#`s from all these comments?

Reply:
Fixed.